apps/node-pl/pattern-lab/_patterns/00-protons-demo/borders.twig

```diff
@@ -94,3 +94,5 @@
     </div>
   </div>
 </div>
+
```

Comment:
Are these intentionally added?

Reply:
If you're asking about the empty line at the end, yep!

Reply:
Roger that. Wasn't used to seeing two empty lines at the end of the file.

Reply:
Whoops.